lost/src/main/java/com/mapzen/android/lost/internal/FusedLocationProviderApiImpl.java

```diff
@@ -164,4 +164,12 @@ public boolean isConnected() {
   public FusedLocationProviderService getService() {
     return service;
   }
+
+  public void removeConnectionCallbacks(LostApiClient.ConnectionCallbacks callbacks) {
```

Comment:
We should make `removeConnectionCallbacks(...)` and `getServiceConnectionManager()` package private since they are only used internally.